rancher/distros-test-framework
# feat: add test runner and makefile adjustments

#### Proposed Changes ####

-feat: add test runner and makefile adjustments

The proposal is to add a test-runner to run make file local and in docker

Bellow you find the related tasks





#### Types of Changes ####
- Yes README and examples updated


#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

- On going. (will add the tests results here )
Create cluster - https://jenkins.int.rancher.io/job/rke2-tests/job/rke2_validate_cluster/345/console


2. Verify code lint; we should not have errors.
- ok 

3. Update the documentation if needed.
- ok

4. Update makefile and docker run if adding new tests.
-ok 

5. Run your tests at least 4 times with all configurations needed and possible.

-ok 
6. If needed test with different os types.
-ok 

#### Linked Issues ####
- https://github.com/rancher/qa-tasks/issues/802
- https://github.com/rancher/qa-tasks/issues/854


#### Further Comments ####
